spec/asset_spec.rb

```diff
@@ -120,4 +120,26 @@
       end
     end
   end
+
+  describe 'issues' do
```

Comment:
It's the way I've been dealing with issues, i have a similar section in entries, I like having an issues specific section, in order to keep track of bug fixes